stages/org.osbuild.coreos.platforms

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/python3
+"""
+Process all necessary operations related to CoreOS
+platforms is crucial and specific to CoreOS. This step
+is essential for CoreOS exclusively.
+"""
+SCHEMA_2 = r"""
+"options": {
+  "additionalProperties": false,
+  "properties": {
+    "platform": {
+      "description": "The artifactory platform name",
+      "type": "string"
+    },
+    "platforms_json": {
+        "description": "The path where platforms.json is stored",
+        "type": "string"
+    },
+    "bootpath": {
+      "type": "string",
+      "description": "The mounted location of the boot filesystem tree where the BLS entries and grubconfig  will be under ./loader/entries/*.conf",
+      "pattern": "^(mount|tree):\/\/\/",
+      "examples": ["tree:///boot", "mount:///", "mount:///boot"],
+      "default": "tree:///boot"
+    },
+    "deployment": {
+      "type": "object",
+      "additionalProperties": false,
+      "required": ["osname", "ref"],
+      "properties": {
+        "osname": {
+          "description": "Name of the stateroot to be used in the deployment",
+          "type": "string"
+        },
+        "ref": {
+          "description": "OStree ref to create and use for deployment",
+          "type": "string"
+        },
+        "serial": {
+          "description": "The deployment serial (usually '0')",
+          "type": "number",
+          "default": 0
+        }
+      }
+    }
+  }
+},
```

Comment:
I feel like we should just not do this here and rather use a `org.osbuild.ostree.deployment` mount here similar to what we do for `org.osbuild.chattr` today: 

https://github.com/osbuild/osbuild/blob/c164665db15603cef768410c9e654226b1369421/test/data/manifests/fedora-coreos-container.mpp.yaml#L267-L290